CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+Please only add new entries to the top of this list, right below the `# Unreleased` header.
+
+# Unreleased
```

Comment:
Also add to this:

"There must always be at least one empty line after all section headers, including `# Unreleased`. This is so that git produces a conflict when a release is made while a PR is open and the PR's changelog entry would go into the wrong section."

Reply:
I realize my explanation [here](https://github.com/rust-windowing/winit/issues/2135#issuecomment-1008907007) might have been a bit misleading on this point - without the newline(s), the diff would look like:
```diff
@@ -1,5 +1,8 @@
 # Unreleased
 
+# Version X
 - Old entry 1
 - Old entry 2
 - Old entry 3
```

Which would still cause a conflict

Reply:
But not in this case:

```
# Unreleased
# Version

- Old entry 1
- Old entry 2
```

Reply:
Oh, you're right! I've added something similar to what you wrote now.